libs/blocks/section-metadata/section-metadata.js

```diff
@@ -0,0 +1,39 @@
+function handleBackground(div) {
+  const pic = div.querySelector('picture');
+  const section = div.closest('.section');
+  if (pic) {
+    if (section) {
+      section.classList.add('has-background');
+      pic.classList.add('section-background');
+      section.insertAdjacentElement('afterbegin', pic);
+    }
+  } else {
+    const color = div.textContent;
+    if (color && section) {
+      section.style.backgroundColor = color;
+    }
+  }
+}
+
+function handleStyle(div) {
+  const value = div.textContent.toLowerCase();
+  const split = value.split(', ');
+  const styles = split.map((style) => style.replaceAll(' ', '-'));
```

Comment:
```suggestion
  const styles = value.split(', ').map((style) => style.replaceAll(' ', '-'));
```
nit: may as well combine these lines